app/assets/src/components/views/PipelineViz/pipeline_viz.scss

```diff
@@ -36,6 +41,7 @@
   overflow: visible;
   position: relative;
   flex-shrink: 0;
+  pointer-events: none;
```

Comment:
what does `pointer-events: none` do?

Reply:
It makes it so that mouse interactions are ignored for the element / go straight to whatever's behind it. I added it in so dragging to pan still works on the spaces between stages